src/osd/sdl/README_SDL20.txt

```diff
@@ -17,15 +17,15 @@ Known bugs:
 * SDL2.0: sdlvideofps does not take -numscreens>1 into account.
 * SDL1.3/WIN32: crashes with -rd d3d
 * SDL1.3/WIN32: resizing does not work
-      
+
 Build SDL 2.0 from HG
 ======================
 
-Pull 2.0 from hg. Than 
+Pull 2.0 from hg. Than
```

Comment:
You might want to correct the then/than confusion as well as removing the trailing whitespace.